packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -412,6 +413,7 @@ export default class Meetings extends WebexPlugin {
       }
 
       return Promise.all([
+        this.webex.internal.services.hasRequiredServices(REQUIRED_SERVICES.postauth, 'postauth'),
```

Comment:
currently all the users are CI registred uses so we will have the utc catalog , Do you know of any type of user not going to get the catalog like guest token etc can you check it